cinema/serializers.py

```diff
@@ -68,13 +79,72 @@ class Meta:
             "movie_title",
             "cinema_hall_name",
             "cinema_hall_capacity",
+            "tickets_available",
+        )
+
+
+class TicketSerializer(serializers.ModelSerializer):
+
+    def validate(self, attrs):
+        data = super(TicketSerializer, self).validate(attrs)
```

Comment:
no need to write `TicketSerializer, self` in super since it will be set there by default